Makefile

```diff
@@ -71,7 +71,7 @@ cleanup-main:
 
 .PHONY: builder
 builder:
-	IMAGE_TAG=main make -C infrastructure/docker/builder build
+	IMAGE_TAG=local make -C infrastructure/docker/builder build
```

Comment:
Why should this value always be local?

Reply:
This target is only used locally, and at other places (e.g. a few lines below in `prepare-compose`) we expect the builder tag to be `local`.